bin/trace

```diff
@@ -320,8 +328,7 @@ sub rewrite_func ($$$$$) {
         next;
       }
       unless ($arg=~/^((\s*[a-z_A-Z](\w|\*)*\s+)+(\s*\*\s*|\s*const\s*|\s*volatile\s*)*)
-              ([a-z_A-Z]\w*)(\[.*?\])?
-              (\s*\/\*\s*(in|out|in_out)\s*\*\/)?\s*$/x) {
+              ([a-z_A-Z]\w*)(\[.*?\])?\s*$/x) {
```

Comment:
Now we just strip out the comments, so there's no need to check for them in the `unless` regex